proto/substrait/algebra.proto

```diff
@@ -91,6 +92,11 @@ message ReadRel {
     repeated Expression.Literal.Struct values = 1;
```

Comment:
Rather than deprecate `VirtualTable` and create `VirtualExpressionTable`, would we consider deprecating the `values` _field_ and adding a new `expressions` field?

Reply:
Expanding on this, I'm almost inclined to have both? Like
```
 message VirtualTable {
    repeated Expression.Literal.Struct values = 1;
    repeated Expression expression = 2;
}
```
which emits first the `values` and then `expressions`. Makes the common case really easy to produce (literals only).

Reply:
I'm not a fan of this. It creates this confusing situation where you have something like 
```
literal1, literal2, expression1, literal3
```
and it gets stored as 

```
values: [literal1, literal2]
expression: [expresion1, literal3]


> Rather than deprecate VirtualTable and create VirtualExpressionTable, would we consider deprecating the values field and adding a new expressions field?

I don't see how that is better (or clearer for someone new to the format).

Reply:
It looks like we have a protobuf check set up so if we can't use the oneof it will let us know.  I do know that it is less permissive than I'm used to:

https://github.com/substrait-io/substrait/blob/main/.github/workflows/pr.yml#L35

Reply:
protobuf oneof can't have repeated fields [link](https://protobuf.dev/programming-guides/proto3/)
`A oneof cannot be repeated.`

Reply:
We can't add oneof as mentioned above so both fields can contain values 
I too share concern with @jacques-n having both fields where initial literals are in Expression_Literal and rest in Expression (assuming there is one expression) doesn't seem intuitive .